runtime/stream.h

```diff
@@ -96,13 +96,13 @@ typedef struct strm_s {
 	BEGINobjInstance;	/* Data to implement generic object - MUST be the first data element! */
 	strmType_t sType;
 	/* descriptive properties */
-	int iCurrFNum;/* current file number (NOT descriptor, but the number in the file name!) */
+	unsigned int iCurrFNum;/* current file number (NOT descriptor, but the number in the file name!) */
```

Comment:
Moving to unsigned is definitely the right thing to do, albeit int should not cause a segfault for negative numbers but rather somewhat strange file names (name-number).